.gitattributes

```diff
@@ -1 +1,4 @@
 *.san filter=lfs diff=lfs merge=lfs -text
+strings.txt working-tree-encoding=cp1255
+/comi/RESOURCE/LANGUAGE.TAB working-tree-encoding=cp1255
+/dig/VIDEO/DIGTXT.TRS working-tree-encoding=cp1255
```

Comment:
Any other files I might have missed?

Reply:
FT has many TRS files, might be better to make it glob pattern `*.TRS`
LANGUAGE.TAB is also used for Grim Fandango if we decide to put it here as well,
and apparently the DIG can LANGUAGE.TAB as well and I would like to check it.
so `LANGUAGE.TAB` also good entry as is

Reply:
Done